dags/utils/gcp.py

```diff
@@ -265,3 +266,88 @@ def reprocess_parquet(parent_dag_name,
             DummyOperator(task_id='no_reprocess')
 
         return dag
+
+
+def export_to_parquet(
+    table,
+    arguments=[],
+    dag_name="export_to_parquet",
+    parent_dag_name=None,
+    default_args=None,
+    aws_conn_id="aws_dev_iam_s3",
+    gcp_conn_id="google_cloud_derived_datasets",
+    dataproc_zone="us-central1-a",
+    dataproc_storage_bucket="moz-fx-data-derived-datasets-parquet",
+    num_preemptible_workers=0,
+):
+
+    """ Export a BigQuery table to Parquet.
+
+    https://github.com/mozilla/bigquery-etl/blob/master/script/pyspark/export_to_parquet.py
+
+    :param str table:                             [Required] BigQuery table name
+    :param List[str] arguments:                   Additional pyspark arguments
+    :param str dag_name:                          Name of DAG
+    :param Optional[str] parent_dag_name:         Parent DAG name
+    :param Optional[Dict[str, Any]] default_args: DAG configuration
+    :param str gcp_conn_id:                       Airflow connection id for GCP access
+    :param str dataproc_storage_bucket:           Dataproc staging GCS bucket
+    :param str dataproc_zone:                     GCP zone to launch dataproc clusters
+    :param int num_preemptible_workers:           Number of Dataproc preemptible workers
+
+    :return: airflow.models.DAG
+    """
+
+    cluster_name = table.replace("_", "-") + "-{{ ds_nodash }}"
```

Comment:
cluster_name needs to be 55 characters or less. This seems fine for `firefox-desktop-exact-mau28-by-dimensions-v1-YYYYMMDD` which ends up being 54 chars but adding a note for future reference.

Reply:
turns out it needs to be `firefox-desktop-exact-mau28-by-dimensions-v1-YYYYMMDD` is 53 characters, and it needs to be under 52 >_<